scripts/delete_databases.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-echo 'Deleting Besu and Orion databases'
+echo 'Deleting Besu, Tessera and Orion databases'
 
 rm -rf workdir/orion1/routerdb workdir/orion1/orion.ports
 rm -rf workdir/orion2/routerdb workdir/orion2/orion.ports
 rm -rf workdir/orion3/routerdb workdir/orion3/orion.ports
```

Comment:
Does Tessera create a ports file as well? We should delete the files created by Tessera.

Reply:
good point. by default it doesn't but we are adding the ability to do this (will require a CLI option to be set)

Reply:
let's leave that change for another PR :)